webapp/src/js/bootstrapper/translator.js

```diff
@@ -8,6 +8,7 @@ const TRANSLATIONS = {
     LOAD_APP: 'Loading app…',
     PURGE_INIT: 'Checking data…',
     PURGE_INFO: ({ count }) => `Cleaned ${count} documents…`,
+    PURGE_META: 'Cleaning meta data…',
```

Comment:
Suggestions?

Reply:
Yeah that's ok. I can't think of a way to explain it that the user's will understand. Leave it for now...